elasticsearch/src/test/scala/com/snowplowanalytics/stream/loader/clients/ElasticsearchBulkSenderSpec.scala

```diff
@@ -31,7 +31,7 @@ import org.specs2.mutable.Specification
 class ElasticsearchBulkSenderSpec extends Specification {
   val node = LocalNode("es", System.getProperty("java.io.tmpdir"))
   node.start()
-  val client       = node.elastic4sclient()
+  val client       = node.client(true)
```

Comment:
there is no `elastic4sclient` anymore and that `true` is for `shutdown after close`